epoch_multinode_test.go

```diff
@@ -246,15 +263,22 @@ type inMemNetwork struct {
 	t         *testing.T
 	nodes     []NodeID
 	instances []*testNode
+	connected map[string]bool
```

Comment:
1. I think it's easier to store the complement - if the node is *not* connected, and let it be connected by default. 
2. Since the test goroutine is asynchronous to the test's communication goroutines, there needs to be some synchronization here. 
3. Let's store empty structs instead of booleans?